### PR TITLE
Constrain certain uses of fragment profile designator to refer to top-level profile (#1038).

### LINKED ISSUE #1038: Constrain <fragment-profile-designator> to reference top-level profile element.
In §8.3.8, constrain `<fragment-profile-designator>` to reference only top-level `tt:profile` elements, which is to say, references are not permitted to nested `tt:profile` elements.

### PR DESCRIPTION
**Pull request by skynavga**
Closes #1038.

### CODE REVIEW

File: spec/ttml2.xml [hunk 1]
```diff
@@ -3119,6 +3119,13 @@ denoting a profile that serves as the baseline profile for this profile.</p>
 <p>If profile is a <loc href="#terms-nesting-profile">nesting profile</loc>, then an ordered list of
 <loc href="#content-value-fragment-profile-designator">&lt;fragment-profile-designator&gt;</loc> expressions, where each designator denotes a
 constituent <loc href="#terms-nested-profile">nested profile</loc>; otherwise, <code>null</code>.</p>
+<note role="explanation">
+<p>If a <loc href="#terms-nested-profile">nested profile</loc> specifies
+an <loc href="#content-attribute-xml-id"><att>xml:id</att></loc> attribute, then that identifer may be used in
```
[the remark this answers is not in the document]
Comment (skynavga) [reply spec/ttml2.xml]: 1. this is a note, so it is intended to be informative, per §2.3;
>All paragraphs marked as a Note are considered non-normative.
2. further, per §2.3, we have (emphasis added)
>**Within normative prose** in this specification, the words may, should, and must are defined as follows...
3. therefore, within a note, _may_, _should_, and _must_ are not conformance keywords
4. we already have many cases where may, should, and must appear in notes and other non-normative text; in particular, examining the first 50 (of ~450) notes, I find 20 uses of may, 2 uses of should, and 3 uses of must;
5. notwithstanding the above, which I believe argues we should not make any change, I will go aheads and replace 'should' with 'is expected to', but don't ask me to remove uses of 'may' from notes.

[the remark this answers is not in the document]
Comment (skynavga) [reply spec/ttml2.xml]: See https://github.com/w3c/ttml2/pull/1165/commits/74b24bee6257fbb8103a822bdf146b9aa06e76a1.

[the remark this answers is not in the document]
Comment (nigelmegitt) [reply spec/ttml2.xml]: > don't ask me to remove uses of 'may' from notes

I didn't. My request was not to add _new_ conformance language.

**Comment by skynavga** [comment]
Tests at w3c/ttml2-tests#235.

**Pull request merged**